roles/lnls-ans-role-sirius-apps/templates/sirius_apps.sh.j2

```diff
@@ -1,61 +1,67 @@
 # EPICS IPs
 # ==================================
 
+# Linac IOCs
+# 10.128.1.12:5064 = LLRF
+# 10.128.1.12:5070 = LLRF (procServControl)
+# 10.128.1.13:5064 = Modulators
+# 10.128.1.13:5068 = Modulators (PC monitoring)
+# 10.128.1.13:5070 = Modulators (procServControl)
+# 10.128.1.54 = BPM 2
+# 10.128.1.55 = BPM 3
+
 IP_IOCS_LI="\
 10.128.1.12:5064 10.128.1.12:5070 \
 10.128.1.13:5064 10.128.1.13:5068 10.128.1.13:5070 \
-10.128.1.54:5064 \
-10.128.1.55:5064 \
+10.128.1.54 \
+10.128.1.55 \
 "
 
-# Undulators CLP device names below are in this order:
-# Ipê (SI-11SP:ID-APU58)
-# Cateretê (SI-07SP:ID-APU22)
-# Manacá (SI-09SA:ID-APU22)
-# Ema (SI-08SB:ID-APU22)
-# Carnaúba (SI-06SB:ID-APU22)
+# Kyma undulators
+# 10.0.38.29 = Ipê (SI-11SP:ID-APU58)
```

Comment:
@edupcoelho , I would keep the ID "serial" name (1995-ccu, 1991a-ccu, ...) here in these comment lines.

Reply:
done!